app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AdminController < ApplicationController
```

Comment:
This brings issues with the templates. How should the views be named in this case?